### DISCUSSION TITLE
Approach to parse HTML link elements within XML string?

### DISCUSSION THREAD

**Discussion by vedtam** [NaturalIntelligence/fast-xml-parser#335] [Q&A]
Hi @amitguptagwl,

I'm fairly new to XML parsing and started out with your parser. It works really nice, however my result has a structure (using the default settings) like:

```
{
    title: 'A/75/845',
    link: 'https://undocs.org/A/75/845',
    description: 'REVISED ESTIMATES ON THE JURISDICTIONAL SET-UP OF THE UNITED ' +
      'NATIONS COMMON SYSTEM: REPORT OF THE SECRETARY-GENERAL<br />[ <a ' +
      'href="https://undocs.org/ar/A/75/845">العربية</a> | <a ' +
      'href="https://undocs.org/zh/A/75/845">中文</a> | <a ' +
      'href="https://undocs.org/en/A/75/845">English</a> | <a ' +
      'href="https://undocs.org/fr/A/75/845">Français</a> | <a ' +
      'href="https://undocs.org/ru/A/75/845">Pусский</a> | <a ' +
      'href="https://undocs.org/sp/A/75/845">Español</a> ]',
    pubDate: 'Thu, 08 Apr 2021 23:55:35 GMT',
    guid: 'A/75/845'
  },
```

I would need to extract here links that contain the word 'en'. Right now I'm doing it via  regex, but it might be not ok in the context of XML / HTML parsing.

Any suggestion on how to further parse `description` so I could extract links as maybe HTML objects in order to avoid using Regex?

Thanks!!

**Comment by amitguptagwl** [comment]
Probably I couldn't understand your problem. What I can understand here is how to extract link with `en` from description property. 

Well! I feel regex is only the better solution to apply on `description` field contents.
